index.html

```diff
@@ -2,9 +2,224 @@
+		c0.1-0.1,0.1-0.2,0.2-0.3c0.1-0.3,0.3-0.5,0.4-0.8c0.1-0.1,0.1-0.2,0.2-0.3c0.1-0.3,0.3-0.5,0.4-0.8c0-0.1,0.1-0.2,0.1-0.3
+		c0.2-0.3,0.3-0.7,0.5-1c0.1-0.3,0.3-0.7,0.4-1c0-0.1,0.1-0.2,0.1-0.3c0.1-0.2,0.2-0.5,0.2-0.7c0-0.1,0.1-0.2,0.1-0.3
+		c0.1-0.2,0.1-0.4,0.2-0.7c0-0.1,0.1-0.2,0.1-0.3c0.1-0.3,0.1-0.6,0.2-0.9c0,0,0,0,0.1,0.1s0.3,0.2,0.5,0.5c1.6,1.5,5.8,6,4.4,12
+		c-0.1,0.5-0.3,1.1-0.5,1.7c-0.1,0.3-0.2,0.5-0.4,0.8c-0.1,0.3-0.3,0.6-0.4,0.8c-0.2,0.3-0.3,0.5-0.5,0.8c0,0,0,0.1-0.1,0.1
+		c-0.2,0.3-0.4,0.6-0.6,0.9c0,0,0,0,0,0.1c-0.2,0.3-0.4,0.5-0.6,0.8l-0.1,0.1c-0.2,0.3-0.5,0.6-0.8,0.9c0,0,0,0-0.1,0.1
+		c-0.2,0.3-0.5,0.5-0.8,0.8c-0.1,0.1-0.1,0.1-0.2,0.2l-1,1c-0.3,0.3-0.6,0.5-0.9,0.8c-0.1,0.1-0.2,0.1-0.3,0.2
+		c-0.4,0.3-0.7,0.6-1.1,0.9l-0.1,0.1c-0.4,0.3-0.8,0.5-1.1,0.8c-0.1,0-0.1,0.1-0.2,0.1l-0.1,0.1c-0.4,0.3-0.9,0.6-1.4,0.9h-0.1
+		c-0.4,0.3-0.9,0.6-1.4,0.8c-0.1,0.1-0.2,0.1-0.3,0.2c0,0-0.1,0-0.1,0.1c0,0,0,0-0.1,0c0.1,0,0.1,0,0.2,0h0.1
+		c0.2-0.1,0.4-0.2,0.6-0.3c0.1,0,0.1-0.1,0.2-0.1s0.1-0.1,0.2-0.1c0.2-0.1,0.3-0.2,0.5-0.3c0.7-0.4,1.4-0.8,2.2-1.2
+		c0,0,0.1,0,0.1-0.1c0.1-0.1,0.2-0.1,0.3-0.2c0.1,0,0.1-0.1,0.2-0.1c0.9-0.5,1.9-1.1,2.9-1.8l0.1-0.1c0.2-0.1,0.4-0.3,0.6-0.4
+		c0.1-0.1,0.2-0.1,0.3-0.2c0.8-0.5,1.6-1.1,2.4-1.6c0.1-0.1,0.3-0.2,0.4-0.3c0.2-0.2,0.5-0.3,0.7-0.5l0.1-0.1
+		c0.5-0.3,0.9-0.7,1.3-1.1l0.1-0.1c0.2-0.2,0.4-0.4,0.6-0.5c0.1-0.1,0.3-0.2,0.4-0.4c0.9-0.8,1.7-1.6,2.4-2.5
+		c0.1-0.1,0.1-0.1,0.2-0.2c0.2-0.3,0.5-0.6,0.7-0.9s0.4-0.6,0.5-0.9c0-0.1,0.1-0.1,0.1-0.2c1.5-0.4,1.7-0.7,1.7-0.9
+		C489,143.8,489,143.9,488.9,144z M552.8,94.1c-7.2,25.6-22.4,26.8-33,24c-0.3-0.5-0.6-1-1-1.5h0.3c6.6,0,18.3-2.9,22.5-22.3
+		c2.9-13.5,7.5-21.4,16.1-27.1c11-7.4,29.1-11,55.1-11c12,0,25.9,0.8,41.5,2.3c25.2,2.5,50.7,6.7,74.5,11.6
+		C627,53.3,561.3,64.1,552.8,94.1z M413.1,37.5l12.1-6.4l12.1,6.4L435.1,24l9.8-9.6l-13.6-2l-6.1-12.3l-6.1,12.3l-13.6,2l9.8,9.6
+		L413.1,37.5z M559.7,363.9l-4.2,20h8.2L559.7,363.9z M486.9,46l4.1,13.1l7.9-11.2l13.7,0.2l-8.2-11l4.4-13l-13,4.4l-11-8.2L485,34
+		l-11.2,7.9L486.9,46z M607.8,567c18.1-1,37.2-17.1,37.5-32.2c-19.8,12.1-30.5,17.4-39.5,26.5C602.1,564.9,603.2,567,607.8,567z
+		 M351.6,47.9l7.9,11.2l4.1-13.1l13.1-4.1L365.5,34l0.2-13.7l-11,8.2l-13-4.4l4.4,13l-8.2,11L351.6,47.9z M468.6,159
+		c0.1-0.1,0.2-0.1,0.4-0.2C468.8,158.9,468.7,158.9,468.6,159L468.6,159z M459.8,372.9h9.2v-13.5h-9.2V372.9z M288.6,192.5
+		c-0.1,0.1-0.2,0.2-0.4,0.3c0.2-0.1,0.4-0.3,0.5-0.3H288.6z M346.5,383.9h8.2l-4-20L346.5,383.9z M442.2,132.2c0-0.2,0-0.4-0.1-0.8
+		c0,0.3,0.1,0.7,0.1,1C442.2,132.4,442.2,132.3,442.2,132.2z M468.2,159.3c0.1-0.1,0.2-0.1,0.3-0.2c0,0,0.1,0,0.1-0.1
+		c-0.4,0.2-0.8,0.4-1.2,0.7C467.5,159.7,467.8,159.5,468.2,159.3z M457.1,139.6c0-0.2,0-0.3,0-0.4l0,0V139.6z"></path>
+</g>
+</svg>
+                        </div>
+                    </a>
+                </div>
+                <div class="social">
+                    <div class="icon">
```

Comment:
Is it buttons for sharing or just links?

Reply:
links